aten/src/TH/generic/THTensorMath.h

```diff
@@ -91,7 +91,11 @@ TH_API void THTensor_(cmin)(THTensor *r, THTensor *t, THTensor *src);
 TH_API void THTensor_(cmaxValue)(THTensor *r, THTensor *t, scalar_t value);
 TH_API void THTensor_(cminValue)(THTensor *r, THTensor *t, scalar_t value);
 
+TH_API void THTensor_(zerosLike)(THTensor *r_, THTensor *input);
```

Comment:
are any changes in this file actually needed?  I got rid of all of them and (if I move the implementation of powOne up in the file) it compiles fine on CPU.